honestcash-v2/src/app/main/components/story-comment-card/story-comment-card.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="card w-100">
+  <div style="padding: 1.25rem;">
+    <div class="row mb-2">
+      <div class="col-1 pr-0">
+        <core-avatar [src]="comment.user && comment.user.imageUrl" large="true"></core-avatar>
+      </div>
+      <div class="col">
+        <div class="row">
+          <div class="col">
+            <a class="font-weight-bold">{{comment.user.username}}</a><br>
+            <span class="text-muted">
+                   {{comment.updatedAt | date:'MMM d, yy'}}
+                 </span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div class="comment-body" [innerHTML]="comment.body"></div>
+      </div>
+    </div>
+
+  </div>
+  <div class="row" style="padding: 0rem 1.25rem 1.25rem 1.25rem;">
+    <main-story-actions></main-story-actions>
+  </div>
+  <main-story-comment-card *ngFor="let subComment of comment.userPosts" [comment]="subComment"></main-story-comment-card>
```

Comment:
this makes comment bubbling (nested) so they appear inside of each other like matrushka dolls. I'll later make this exactly same as v1 where it is padded on the left side on each nesting level